src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,88 @@
+import { useSearchParams, Link } from 'react-router-dom';
+import cn from 'classnames';
+
+const SexFilter:{ [key:string]:string } = {
+  All: '',
+  Male: 'm',
+  Female: 'f',
+};
+
+const CenturyFilter = [16, 17, 18, 19, 20];
+
 export const PeopleFilters = () => {
+  // const [centuries, setCenturies] = useState<CenturyFilter[]>([]);
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const handleSexChange = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    type: string,
+  ) => {
+    event.preventDefault();
+    const params = new URLSearchParams(searchParams);
+
+    if (type === 'All') {
```

Comment:
Use Enum instead of the hardcoded values

Reply:
I tried enum in the first place, but it gives me [this](https://clip2net.com/s/4jNlHxl) error whenever I address the value of enum through a variable as a key. What am I missing?